src/layer_normalization.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+import {add, sub, mul, div, pow} from './binary.js';
+import {reduceMean} from './reduce.js';
+import {reshape} from './reshape.js';
+import {Tensor, Scalar} from './lib/tensor.js';
+import {validateLayerNormalizationParams} from './lib/validate-input.js';
+
+/**
+ * Normalize the tensor values of input features using
+ * [layer-Normalization]
+ * @param {Tensor} input
+ * @param {Array} axes
+ * @param {MLBatchNormalizationOptions} [options]
+ * @return {Tensor}
+ */
+export function layerNormalization(input, {scale, bias, axes, epsilon=1e-5}) {
+  validateLayerNormalizationParams(...arguments);
+  console.log('axes :', axes);
+  if (axes === undefined) {
```

Comment:
Synced with @shiyi9801, here default **axes** should be [1, ..., N-1] which N means the rank of input.